src/gtp_packet.erl

```diff
@@ -1644,8 +1644,10 @@ decode_v1_element(<<M_flags_dual_address_bearer_flag:1,
     #common_flags{instance = Instance,
 		  flags = [ 'Dual Address Bearer Flag' || M_flags_dual_address_bearer_flag =/= 0 ] ++ [ 'Upgrade QoS Supported' || M_flags_upgrade_qos_supported =/= 0 ] ++ [ 'NRSN' || M_flags_nrsn =/= 0 ] ++ [ 'No QoS negotiation' || M_flags_no_qos_negotiation =/= 0 ] ++ [ 'MBMS Counting Information' || M_flags_mbms_counting_information =/= 0 ] ++ [ 'RAN Procedures Ready' || M_flags_ran_procedures_ready =/= 0 ] ++ [ 'MBMS Service Type' || M_flags_mbms_service_type =/= 0 ] ++ [ 'Prohibit Payload Compression' || M_flags_prohibit_payload_compression =/= 0 ]};
 
-decode_v1_element(<<>>, 149, Instance) ->
-    #apn_restriction{instance = Instance};
+decode_v1_element(<<M_restriction_type_value:8/integer,
+	_/binary>>, 149, Instance) ->
+	#apn_restriction{instance = Instance,
+		restriction_type_value = M_restriction_type_value};
```

Comment:
That part of the code is auto generated by priv/ie_gen_v1.erl. You have to adjust the declaration there and regenerate the code.

Reply:
I updated the file and regenerated the code

Reply:
And please also adjust the IE test in test/property_test/gtplib_prop.erl

Reply:
Done

===> Running Common Test suites...
%%% gtp_SUITE: ...........
All 11 tests passed.